scripts/cluster-helpers.sh

```diff
@@ -139,3 +102,173 @@ function native_terminate() {
 	echo
 
 }
+
+###############################  v MiniKube v #################################
+
+function minikube_start() {
+	echo
+	echo "###   Installing hpo for minikube"
+	echo
+
+	# If hpo_ns was not set by the user
+	if [ -z "$hpo_ns" ]; then
+		hpo_ns="monitoring"
+	fi
+
+	minikube_first
+	minikube_deploy
+}
+
+function minikube_first() {
+
+	kubectl_cmd="kubectl -n ${hpo_ns}"
+	echo "Info: One time setup - Create a service account to deploy hpo"
+	
+	${kubectl_cmd} apply -f ${HPO_SA_MANIFEST}
+	check_err "Error: Failed to create service account and RBAC"
+
+	${kubectl_cmd} apply -f ${HPO_ROLE_MANIFEST}
+	check_err "Error: Failed to create role"
+
+	sed -e "s|{{ HPO_NAMESPACE }}|${hpo_ns}|" ${HPO_RB_MANIFEST_TEMPLATE} > ${HPO_RB_MANIFEST}
+	${kubectl_cmd} apply -f ${HPO_RB_MANIFEST}
+	check_err "Error: Failed to create role binding"
+}
+
+# You can deploy using kubectl
+function minikube_deploy() {
+
+	echo "Info: Deploying hpo yaml to minikube cluster"
+
+	# Replace hpo docker image in deployment yaml
+	sed -i "s|{{ HPO_IMAGE }}|${HPO_CONTAINER_IMAGE}|" ${HPO_DEPLOY_MANIFEST}
```

Comment:
Use the deployment template to find and replace and then redirect it to another file like below
sed -i "s|{{ HPO_IMAGE }}|${HPO_CONTAINER_IMAGE}|" ${HPO_DEPLOY_MANIFEST_TEMPLATE} > ${HPO_DEPLOY_MANIFEST}

Reply:
Done